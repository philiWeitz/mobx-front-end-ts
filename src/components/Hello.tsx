
import * as React from 'react';
import { transNameSpace } from '../util/i18n';

const t = transNameSpace('other-name-space');


export default class Hello extends React.Component {
  render() {
    return (
      <div>
        <h1>Hello!</h1>
        <p>{t('hello')}</p>
      </div>
    );
  }
}
