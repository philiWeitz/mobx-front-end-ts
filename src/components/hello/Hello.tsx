
import * as React from 'react';
import * as style from './styles.scss';

export default class Hello extends React.Component {
  render() {
    return (
      <div className={style.page}>
        <h1>Hello World Component!</h1>
        <p>hello</p>
      </div>
    );
  }
}
