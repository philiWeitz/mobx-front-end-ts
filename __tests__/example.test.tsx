/* global describe, it, expect */

import * as React from "react";
import { shallow } from "enzyme";
import enzymeToJson from 'enzyme-to-json';

import Hello from '../src/components/Hello';

describe('example test', () => {

  it('should run a TS test', () => {
    expect(1).toBe(1);
  });

  it('should perform a snapshot test', () => {
    const wrapper = shallow(
      <Hello />,
    );
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  })

});