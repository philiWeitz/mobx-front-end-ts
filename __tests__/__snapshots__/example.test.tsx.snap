// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example test should perform a snapshot test 1`] = `
<div>
  <h1>
    Hello!
  </h1>
  <p />
</div>
`;
